From 1427d5a66edec516359557bccc1315a34bbf5067 Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Mon, 19 Nov 2012 15:23:56 +0200
Subject: [PORT FROM MAIN] Telephony: Increase the Send SMS retry delay timer

BZ: 22238

When the device is registered to the network,framework will
start the SMS retry. Number of retry is 3 and the retry interval
is 2seconds. Since the retry interval is very less, some of the
SMS are never sent out. Failure may be due to modem busy with SIM
files caching or service center busy etc.

This patch increases the SEND_RETRY_DELAY to 10seconds.

Change-Id: I70160ca6a37cfdfb1c94ac39f5fd31f24ab249a7
Orig-Change-Id: Iee8c8ee2b06ccf3a8a5dbba4edb65667734f36a2
Signed-off-by: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
Reviewed-on: http://android.intel.com:8080/75573
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Fremaux, Pascal <pascal.fremaux@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 .../android/internal/telephony/SMSDispatcher.java  |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/java/com/android/internal/telephony/SMSDispatcher.java b/src/java/com/android/internal/telephony/SMSDispatcher.java
index b02e24b..b4c0419 100644
--- a/src/java/com/android/internal/telephony/SMSDispatcher.java
+++ b/src/java/com/android/internal/telephony/SMSDispatcher.java
@@ -159,7 +159,7 @@ public abstract class SMSDispatcher extends Handler {
     /** Maximum number of times to retry sending a failed SMS. */
     private static final int MAX_SEND_RETRIES = 3;
     /** Delay before next send attempt on a failed SMS, in milliseconds. */
-    private static final int SEND_RETRY_DELAY = 2000;
+    private static final int SEND_RETRY_DELAY = 10000;
     /** single part SMS */
     private static final int SINGLE_PART_SMS = 1;
     /** Message sending queue limit */
-- 
1.7.4.1

